storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/scheduling/TestConstraintSolverStrategy.java

```diff
@@ -159,12 +316,30 @@ public void testConstraintSolverForceBacktrack() {
                 return super.sortByValues(map).descendingMap();
             }
         };
-        basicUnitTestWithKillAndRecover(cs, BACKTRACK_BOLT_PARALLEL);
+        basicUnitTestWithKillAndRecover(cs, BACKTRACK_BOLT_PARALLEL, 2);
+    }
+
+    @Test
+    public void testConstraintSolverForceBacktrackWithSpreadCoLocation() {
```

Comment:
What's the different between this and `testConstraintSolverForceBacktrack `

Reply:
None. Removed testConstraintSolverForceBacktrack().

Reply:
Also added Assertion to check for invalid test case (i.e. when maxCoLocationCnt is set to > 1 and testing under old config scenario.